projects/fsastorefrontlib/src/core/my-account/store/reducers/quote.reducer.ts

```diff
@@ -12,6 +13,13 @@ export function reducer(
   action: fromAction.QuoteAction
 ): QuoteState {
   switch (action.type) {
+    case fromAction.LOAD_QUOTES:
+    case fromAction.LOAD_QUOTE_COMPARISON:
+    case fromAction.LOAD_QUOTE_DETAILS: {
```

Comment:
On every load action, we reset the store state to the initial state. This solves the issue on the details page e.g. quote when with every new visit of quote details first was emitted last visited quote and after few seconds comes data for a new quote.